### PR TITLE
attempt to check AASM compatible with latest rubies

### PR DESCRIPTION
**Pull request by ferdinandrosario**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,12 +7,12 @@ rvm:
   # - 1.9.2
   - 1.9.3
   - 2.0.0
-  - 2.1
-  - 2.2
+  - 2.1.7
+  - 2.2.3
```
Comment (alto) [inline .travis.yml]: Aren't `2.1` and `2.2` picking the latest version available for `rvm`?

File: .travis.yml [hunk 2]
```diff
@@ -7,12 +7,12 @@ rvm:
...
   # - jruby-18mode # JRuby in 1.8 mode
   - jruby-1.7 # JRuby in 1.9 mode
-  - jruby-9.0.3.0
-  - rbx-2.2.1
+  - jruby-9.0.4.0
+  - rbx-2.5.8
```
Comment (alto) [inline .travis.yml]: I wonder if there's something available for _JRuby_ and _Rubinius_ as well.

**Comment by ferdinandrosario** [comment]
yes I check in the detail page. so that I have changed to specific patch version...

**Comment by ferdinandrosario** [comment]
I think jruby 1.7 also passing but it's under allowed failure. can we change that too?

**Comment by alto** [comment]
Yes, `jruby-1.7` can be removed from allowed failures now. I left it there for a while because we had some troubles in the past. But it should be stable now.

**Comment by ferdinandrosario** [comment]
also thinking of update all rails version for the latest stable. my motivation is to encourage people to update corresponding patch version so that migrate won't hurt. Can i do that? what's your opinion

**Comment by alto** [comment]
Sure, just go ahead.

**Comment by alto** [comment]
So keep on using `2.1` and `2.2`?

**Comment by ferdinandrosario** [comment]
ok.I will keep 2.1 & 2.2

**Comment by alto** [comment]
Yay! ;)

**Comment by ferdinandrosario** [comment]
$ ruby --version
ruby 2.1.5p273 (2014-11-13 revision 48405) [x86_64-linux]
this is the version trvis taking while CI.
shall I change?
I we costing too much for travis :laughing:

**Comment by alto** [comment]
@ferdinandrosario Let's try get this out of the door. Could you please merge back _master_ into your branch and remove the conflicts? I will merge afterwards.

**Comment by ferdinandrosario** [comment]
done :+1:

**Comment by alto** [comment]
Lovely. Let's wait for the build and we are ready to go :+1:

**Comment by alto** [comment]
Yay!

**Comment by alto** [comment]
Thanks a lot. I really appreciate your work.

**Comment by ferdinandrosario** [comment]
:smiley:

**Comment by ferdinandrosario** [comment]
thank u so much for all patient comments.

**Pull request merged**